doc/rbd/rbd-nomad.rst

```diff
@@ -52,20 +52,24 @@ diagram depicts the Nomad/Ceph technology stack.
 Create a Pool
 =============
 
-By default, Ceph block devices use the ``rbd`` pool. Create a pool for
-Nopmad persistent storage. Ensure your Ceph cluster is running, then create
-the pool. ::
+By default, Ceph block devices use the ``rbd`` pool. Create a pool for Nomad
+persistent storage. Ensure that your Ceph cluster is running, then create the
+pool:
```

Comment:
```suggestion
By default, Ceph block devices use the ``rbd`` pool. Ensure that your Ceph
cluster is running, then create a pool for Nomad persistent storage:
```

Reply:
This suggestion has been incorporated into the text.